api/apps/api/src/modules/projects-acl/project-acl.service.ts

```diff
@@ -5,26 +5,89 @@ import { intersection } from 'lodash';
 import { UsersProjectsApiEntity } from '@marxan-api/modules/projects/control-level/users-projects.api.entity';
 import { Roles } from '@marxan-api/modules/users/role.api.entity';
 import { Injectable } from '@nestjs/common';
+import {
+  AccessControlService,
+  Permit,
+} from '../access-control/access-control-service.interface';
 
 /**
  * Debt: neither UsersProjectsApiEntity should belong to projects
  * nor the Roles should belong to users
  */
 @Injectable()
-export class ProjectAclService {
-  private readonly canPublishRoles = [
-    Roles.project_admin,
+export class ProjectAclService extends AccessControlService {
+  private readonly canCreateScenarioRoles = [Roles.project_owner];
+  private readonly canEditScenarioRoles = [Roles.project_owner];
+  private readonly canViewSolutionRoles = [Roles.project_owner];
+  private readonly canPublishProjectRoles = [Roles.project_owner];
+  private readonly canViewProjectRoles = [
     Roles.project_owner,
-    Roles.organization_admin,
-    Roles.organization_owner,
+    Roles.project_admin,
+    Roles.project_user,
   ];
 
   constructor(
     @InjectRepository(UsersProjectsApiEntity)
     private readonly roles: Repository<UsersProjectsApiEntity>,
-  ) {}
+  ) {
+    super();
+  }
+  // TODO: create a proper workflow to check project creation capabilities
```

Comment:
```suggestion
  // TODO: this will be changed in the following release of user requirements.
  // For now, anyone should be able to create projects, regardless of having roles or not.
  // In the future project creation will be limited to organization contributors, so this logic will be moved to the access control module
```